### PR TITLE
Improve Custom Views docs

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Improve Custom Views docs

#### Description

There are two main changes:
1. We rename all references to the configuration file name `custom-view-config.json` to `custom-view-config.mjs` as that's the one we use in our starter templates
2. We add the docs for the `env.development.hostUriPath` config property.

### CODE REVIEW

**Review by ddouglasz — APPROVED** [review]
Thank you 🙏🏾

**Review by emmenko — APPROVED** [review]
Thanks

File: website-custom-views/src/content/api-reference/custom-view-config.mdx [hunk 1]
```diff
@@ -174,6 +174,22 @@ If you need to apply more granular permissions for local development of a Custom
 
 For more information, see [Granular permissions for local development](/help-needed#granular-permissions-for-local-development).
 
+## `env.development.hostUriPath`
+
+The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/<custom-view-host>` by default.
+
+To test your Custom View in a specific built-in application, you can override the default behavior to provide a custom URI path by setting the `env.development.hostUriPath`.
+
+```json
```
Comment (Anshuman71) [inline website-custom-views/src/content/api-reference/custom-view-config.mdx:183]: Hi @CarlosCortizasCT double-check the path now. I added the `projectKey` to it.

File: website-custom-views/src/content/api-reference/custom-view-config.mdx [hunk 2]
```diff
@@ -174,6 +174,22 @@ If you need to apply more granular permissions for local development of a Custom
 
 For more information, see [Granular permissions for local development](/help-needed#granular-permissions-for-local-development).
 
+## `env.development.hostUriPath`
+
+The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/<custom-view-host>` by default.
```
Comment (Anshuman71) [inline website-custom-views/src/content/api-reference/custom-view-config.mdx:179]: @CarlosCortizasCT please check the changes I made 93c96eb60a478ad545ad6850650b801620f6d3b8 if the meaning is still intact.

**Review by Anshuman71 — APPROVED** [review]
@CarlosCortizasCT please see my updates and comments before merging. Looks good otherwise.

File: website-custom-views/src/content/api-reference/custom-view-config.mdx [hunk 3]
```diff
@@ -176,17 +176,15 @@ For more information, see [Granular permissions for local development](/help-nee
 
 ## `env.development.hostUriPath`
 
-The Custom View [context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) will contain the ["hostUrl"](https://docs-beta-custom-views.commercetools.vercel.app/custom-views/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property which holds the value of the host built-in application view where the user opens it.
+The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/<custom-view-host>` by default.
```
Comment (emmenko) [inline website-custom-views/src/content/api-reference/custom-view-config.mdx:179]: The value the actually `@@custom-view-host@@`

```suggestion
The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/@@custom-view-host@@` by default.
```

File: website-custom-views/src/content/api-reference/custom-view-config.mdx [hunk 4]
```diff
@@ -176,17 +176,15 @@ For more information, see [Granular permissions for local development](/help-nee
...
 
-When working locally, by default you will see the `hostUrl` value pointing to local enviroment: `http://localhost:3001/<projectKey>/@@custom-view-host@@`.
-
-You can override this default behaviour to set a custom URI path so you can test the Custom View in the context of a specific built-in application view.
+To test your Custom View in a specific built-in application, you can override the default behavior to provide a custom URI path by setting the `env.development.hostUriPath`.
```
Comment (emmenko) [inline website-custom-views/src/content/api-reference/custom-view-config.mdx:181]:
```suggestion
To simulate your Custom View as if it's rendered within a specific built-in application, you can override the default behavior to provide a custom URI path by setting the `env.development.hostUriPath`.
```

File: website-custom-views/src/content/concepts/oauth-scopes-and-user-permissions.mdx [hunk 5]
```diff
@@ -13,7 +13,7 @@ To fulfill these requirements, your Custom View would need the following OAuth S
 
 These [OAuth Scopes](https://docs-beta-custom-views.commercetools.vercel.app/api/scopes) must be specified in your Custom View config, using the [`oAuthScopes` field](/api-reference/custom-view-config#oauthscopes).
 
-```json title="custom-view-config.mjs"
+```js title="custom-view-config.mjs"
```
Comment (emmenko) [inline website-custom-views/src/content/concepts/oauth-scopes-and-user-permissions.mdx:16]: Nit: hmm if we set the syntax to JS, we should probably adjust the content to be JS syntax and not JSON.

**Review by emmenko — COMMENTED** [review]
Thanks for the suggestions!

File: website-custom-views/src/content/api-reference/custom-view-config.mdx (hunk 3 above)
Comment (Anshuman71) [reply website-custom-views/src/content/api-reference/custom-view-config.mdx:179]: Thanks @emmenko. Can you share why we use this format specifically @@?

Comment (emmenko) [reply website-custom-views/src/content/api-reference/custom-view-config.mdx]: It's the actual value, it's not a placeholder.

This is essentially the equivalent of Custom Applications `entryPointUriPath`. However, Custom Views don't need an entry point but to keep using the same logic and infrastructure as for Custom Applications we decided to define a special entry point value for Custom Views.
That's why it has this special format.

Comment (Anshuman71) [reply website-custom-views/src/content/api-reference/custom-view-config.mdx]: Got it. Thanks, Nicola for the explanation.

File: website-custom-views/src/content/concepts/oauth-scopes-and-user-permissions.mdx (hunk 5 above)
Comment (Anshuman71) [reply website-custom-views/src/content/concepts/oauth-scopes-and-user-permissions.mdx:16]: Hi @emmenko, thanks. I’ve applied this change too all .mjs files in 4c0aefbc6c898033694c442f3d7af3bfe0944430

**Review by emmenko — APPROVED** [review]
Thanks!

**Pull request merged**